app/controllers/users_controller.rb

```diff
@@ -35,4 +35,60 @@ def upload_avatar
       render 'edit'
     end
   end
+
+  # POST /user/update_email
+  def send_update_email_token_email
```

Comment:
Convert this into a GraphQL request

Reply:
@developedBySJ Why was this resolved with the POST request code still in place? 😕

Reply:
@developedBySJ  I will clean up the code.